src/templates/little_machine/item.html

```diff
@@ -1,12 +1,11 @@
-<li class="little-machine-item result">
-  <span class="icon icon-{{model.type}}"></span>
-  <img class="icon favicon" src="{{model.faviconUrl}}">
-  <span class="result-copy">
-    {{#model.title}}
-      {{{model.title}}} &mdash;
-    {{/model.title}}
-    <span class="result-url">
-      {{model.url}}
-    </span>
+<li class="activity-item result result--little-machine">
+  <span class="icon icon--recommendation"></span>
+  <span class="result--segment result--segment--little-machine">
+    <figure class="little-machine--segment little-machine--image">
+      <img src="{{model.image.url}}" width="64" height="64"/>
```

Comment:
Do we have feelings on hard coding `width` and `height` in HTML markup instead of CSS?